packages/starknet-snap/src/rpcs/estimateFee.test.ts

```diff
@@ -89,7 +116,9 @@ describe('estimateFee', () => {
         version: constants.TRANSACTION_VERSION.V1,
       },
     );
-    expect(result).toStrictEqual(estimateBulkFeeRespMock);
+    const { estimateResults, ...resp } = estimateBulkFeeRespMock;
+
+    expect(result).toStrictEqual(resp);
```

Comment:
nit
```suggestion
    expect(result).toStrictEqual({
      includeDeploy: estimateBulkFeeRespMock.includeDeploy,
      overallFee: estimateBulkFeeRespMock.overallFee,
      suggestedMaxFee: estimateBulkFeeRespMock.suggestedMaxFee,
      unit: estimateBulkFeeRespMock.unit,
    });
```

that may be more specify, wdyt?